packages/ui/lit-spreadsheet/src/position.ts

```diff
@@ -0,0 +1,80 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+export type CellPosition = { i: number; j: number };
```

Comment:
i'm glad we're lifting these definition. bring the tests also?

Reply:
I haven’t synced this implementation with yours yet, that should probably come as a follow-up.